### DISCUSSION TITLE
Project longevity

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/finops-toolkit#724] [General]
Hello, I've been looking at this product and whether or not it's something we would like to implement within our company. I know the project is quite new at this point, is this something you're expecting to continue developing and expanding for the long term?

**Comment by flanakin** [comment]
Yes! We have a large backlog of features we'd like to add. Check out the [roadmap](https://github.com/microsoft/finops-toolkit/wiki/Roadmap) to get an idea of where we're headed and let us know if there's anything you'd like to see prioritized or added to the list!
